front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -50,6 +49,28 @@ export class AnalysisPage extends Component {
           >
             <div className="flex gap-y-10 w-full flex-col h-full">
               <div className= "w-full bg-gray-100 pb-2">
+                {
+                    !this.state.showIntentMenu &&
+                    <button
+                        className="absolute mx-5 my-2"
+                        onClick={this.openIntentMenu}
+                        aria-label="intent menu button"
+                        data-testid="intent-menu-button"
+                    >
+                        <label>
+                            <svg
+                                xmlns="http://www.w3.org/2000/svg"
+                                width="50"
+                                height="60"
+                                viewBox="2 0 30 30"
+                            >
+                                <path d="M26,8c0,1.104-0.896,2-2,2H8c-1.104,0-2-0.896-2-2s0.896-2,2-2h16C25.104,6,26,6.896,26,8z"/>
+                                <path d="M26,16c0,1.104-0.896,2-2,2H8c-1.104,0-2-0.896-2-2s0.896-2,2-2h16C25.104,14,26,14.896,26,16z"/>
+                                <path d="M26,24c0,1.104-0.896,2-2,2H8c-1.104,0-2-0.896-2-2s0.896-2,2-2h16C25.104,22,26,22.896,26,24z"/>
+                            </svg>
+                        </label>
```

Comment:
Also I think we should reduce the clutter on this svg by extracting it + we should give it a title for a11y (i.e. "open navigation menu").

Reply:
+1 on this, you can probably do the same method previously where the svg was it's own file and then import it as a react component

Reply:
What do you mean by add a title? Like add a tooltip or add another label?

Reply:
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title